language/de-DE/mod_articles.ini

```
@@ -16,7 +16,7 @@ MOD_ARTICLES_FIELD_AUTHOR_LABEL="Autoren"
 MOD_ARTICLES_FIELD_CATDEPTH_LABEL="Kategorietiefe"
 MOD_ARTICLES_FIELD_CATFILTERINGTYPE_DESC="Die ausgewählten Kategorien ein- oder ausschließen."
 MOD_ARTICLES_FIELD_CATFILTERINGTYPE_LABEL="Kategorien-Filtertyp"
-MOD_ARTICLES_FIELD_COUNT_DESC="Die anzuzeigende Anzahl an Einträgen.<br><br><strong>Hinweis:</strong><br>Der Standardwert „0“ zeigt alle Beiträge an."
+MOD_ARTICLES_FIELD_COUNT_DESC="Die Anzahl der anzuzeigenden Einträge. Auf „0“ setzen, um alle Einträge anzuzeigen."
```

```suggestion
MOD_ARTICLES_FIELD_COUNT_DESC="Die Anzahl der anzuzeigenden Einträge. Der Standardwert „0“ zeigt alle Einträge an."
```
Passt es hier vielleicht besser?